packages/portal-proto/src/components/QuickSearch/QuickSearch.tsx

```diff
@@ -96,56 +106,51 @@ export const QuickSearch = (): JSX.Element => {
         ? `File ${matchingToken} has been updated`
         : matchingToken;
 
+      const IconFormatted = ({ Icon }: { Icon: any }): JSX.Element => (
+        <div className="bg-accent-cool-content-lighter rounded-full">
+          <Icon className="p-1.5 w-10 h-10" aria-hidden />
+        </div>
+      );
+      const entityForMapping = entity || atob(label).split(":")[0];
       return (
         <div
           data-testid="text-search-result"
           ref={ref}
           {...others}
-          aria-label={`${badgeText}, ${matchingToken}`}
+          aria-label={`${badgeText}, ${matchingToken}, Category: ${entityForMapping}`}
         >
           <div
-            className={`flex p-2 px-4 ${
-              others["data-hovered"] &&
-              "bg-primary-darkest text-primary-contrast-darkest"
+            className={`px-4 ${
+              others["data-hovered"] ? "bg-primary-lightest" : ""
             }`}
           >
-            <div className="self-center">
-              <TypeIcon
-                iconText={
-                  entityShortNameMapping[entity || atob(label).split(":")[0]]
-                }
-                changeOnHover={others["data-hovered"]}
-              />
-            </div>
-            <div className="flex flex-col">
-              <div style={{ width: 200 }}>
-                <Badge
-                  classNames={{
-                    inner: "text-xs",
-                    root: `${
-                      others["data-hovered"]
-                        ? "bg-primary-contrast-darker text-primary-darker"
-                        : "bg-primary-darker text-primary-contrast-darker"
-                    }`,
-                  }}
-                  className="cursor-pointer"
-                >
-                  {badgeText}
-                </Badge>
+            <div
+              className={`py-2 flex gap-2 ${
+                last ? "" : "border-b border-gdc-grey-light"
```

Comment:
Nitpick: Do you think whether we can leverage `last` pseudo class here: https://tailwindcss.com/docs/hover-focus-and-other-states#first-last-odd-and-even?

Reply:
the problem is its a sub element, do you know if there is a way to handle for that?